didomi/flutter
# Update native sdks + update tests

Update native SDKs

Fix UI test with latest Native SDK
- Change iOS invalid key behavior
- Update Notice tests (iOS is requiring `setupUI` to get the view controller)
- Update clearUser() usage for Android (requires initialization) - Skipped for Android, will be reverted after the next native SDK update
